lib/buckets.ps1

```diff
@@ -117,6 +117,8 @@ function Convert-RepositoryUri {
             $Matches.provider, $Matches.user, $Matches.repo -join '/'
         } else {
             error "$Uri is not a valid Git URL!"
+            error "Please see https://git-scm.com/docs/git-clone#_git_urls for valid ones."
+            exit 1
```

Comment:
Exiting from script might not be desired if we use this function somewhere else. Instead we can check something like last error code or return value of function.